Makefile

```diff
@@ -24,7 +24,8 @@ usage:
 	@echo " emulation-check: Run the emulation tests for supported boards"
 	@echo "           clean: Clean all builds"
 	@echo "          format: Runs the rustfmt tool on all kernel sources"
-	@echo "       formatall: Runs all formatting tools"
+	@echo "          clippy: Runs the clippy code linter on all kernel sources"
```

Comment:
Ah! I missed this when emulation-* came in, but this list used to be in alpha-order; can you fix emulation and put clippy in the right place too?